source/features/code-highlight.js

```diff
@@ -0,0 +1,43 @@
+import prism from "prismjs";
+import 'prismjs/components/prism-jsx'
+import 'prismjs/components/prism-bash'
+import 'prismjs/components/prism-git'
+import 'prismjs/components/prism-typescript'
+import 'prismjs/components/prism-scss'
+
+function isSupportedLanguage(lang) {
+	const supportedLang = ["javascript", "jsx", "bash", "git", "typescript", "html", "css", "scss"]
+	return supportedLang.includes(lang);
+}
+
+export default function() {
+	const postsContent = $(".stream .stream-items > li .js-tweet-text-container > p");
+
+	postsContent.each((i,el) => {
+		const regex = /```(\w.*)([^```]+)```/g;
+		const postContent = $(el).text();
+		const capturingGroup = regex.exec(postContent);
+
+		if (capturingGroup && capturingGroup.length === 3) {
+			const code = capturingGroup[2].replace(/↵/g, "");
+			const selectedLang = capturingGroup[1].toLowerCase();
+
+			if (isSupportedLanguage(selectedLang)) {
+				const highlightedCode = prism.highlight(code, prism.languages[selectedLang]);
+				const updatedHtml = `
+					<pre class='language-${selectedLang}'>
+						<div class='RT_highlight__lang'>
+						${selectedLang}
+						</div>
+						<code class='language-${selectedLang}'>
+							<div>${highlightedCode}</div>
+						</code>
+					</pre>`
```

Comment:
You can use JSX here. Example: https://github.com/sindresorhus/refined-github/blob/d10510f660d4e1393ad01af097f97de36ced680c/source/features/add-delete-fork-link.js#L17-L19

Just make sure you import `dom-chef` in files you use it: `import {h} from 'dom-chef';`

The benefit of using JSX here is syntax highlighting, performance, and XSS prevention.